app/components/repository-layout.js

```diff
@@ -17,10 +22,30 @@ export default Ember.Component.extend({
     return this.get('externalLinks').githubRepo(slug);
   },
 
+  @computed('repo.id', 'repo.defaultBranch.name')
```

Comment:
good catch! I think I can actually get if from the repo model as well and remove this completely!